### PR TITLE
Classification properties

### PR DESCRIPTION
**Pull request by l2280212**
This PR reorganizes the iotdb-system.properties.template and iotdb-system.properties files, making the iotdb-system.properties contains more important parameters.

### CODE REVIEW

File: iotdb-core/node-commons/pom.xml [hunk 1]
```diff
@@ -188,7 +188,7 @@
             <resource>
                 <directory>${project.basedir}/src/assembly/resources/conf</directory>
                 <includes>
-                    <include>iotdb-system.properties</include>
+                    <include>iotdb-system.properties.template</include>
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/node-commons/pom.xml:191]: fix ConfigurationFileUtils

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template [hunk 2]
```diff
@@ -0,0 +1,1874 @@
...
+
+####################
+### Node RPC Configuration
+####################
+
+# Used for RPC communication inside cluster.
+# Could set 127.0.0.1(for local test) or ipv4 address.
+# effectiveMode: first_start
+# Datatype: String
+cn_internal_address=127.0.0.1
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template:37]: we can comment all parameter now for consistency

Comment (l2280212) [reply iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template:58]: fix

File: iotdb-core/node-commons/pom.xml (hunk 1 above)
Comment (l2280212) [reply iotdb-core/node-commons/pom.xml:191]: fix

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template (hunk 2 above)
Comment (l2280212) [reply iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template:58]: fix

File: iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template [hunk 3]
```diff
@@ -0,0 +1,1868 @@
...
+# effectiveMode: hot_reload
+compressor=LZ4
+
+####################
+### Authorization Configuration
+####################
+
+# which class to serve for authorization. By default, it is LocalFileAuthorizer.
+# Another choice is org.apache.iotdb.commons.auth.authorizer.OpenIdAuthorizer
+authorizer_provider_class=org.apache.iotdb.commons.auth.authorizer.LocalFileAuthorizer
```
Comment (shuwenwei) [inline iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template:1485]: \# effectiveMode: restart

Comment (l2280212) [reply iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template:1486]: fix it

**Pull request merged**